python_modules/libraries/dagster-teradata/dagster_teradata/resources.py

```diff
@@ -0,0 +1,464 @@
+                    teradata.execute_queries(
+                        sql_queries=queries
+                    )
+
+        """
+        check.sequence_param(sql_queries, "sql_queries", of_type=str)
+
+        results: List[Any] = []
+        with self.get_connection() as conn:
+            with closing(conn.cursor()) as cursor:
+                for sql in sql_queries:
+                    self.log.info("Executing query: " + sql)
+                    cursor.execute(sql)
+                    if fetch_results:
+                        results = results.append(cursor.fetchall())  # type: ignore
+                        return results
+
+    @public
+    def s3_to_teradata(
+        self,
+        s3: S3Resource,
+        s3_source_key: str,
+        teradata_table: str,
+        public_bucket: bool = False,
+        teradata_authorization_name: str = "",
+    ):
+        """Loads CSV, JSON and Parquet format data from Amazon S3 to Teradata.
+
+        Args:
+            :param s3_source_key: The URI format specifying the location of the S3 bucket.(templated)
+                The URI format is /s3/YOUR-BUCKET.s3.amazonaws.com/YOUR-BUCKET-NAME.
+                Refer to
+                https://docs.teradata.com/search/documents?query=native+object+store&sort=last_update&virtual-field=title_only&content-lang=en-US
+            :param public_bucket: Specifies whether the provided S3 bucket is public. If the bucket is public,
+                it means that anyone can access the objects within it via a URL without requiring authentication.
+                If the bucket is private and authentication is not provided, the operator will throw an exception.
+            :param teradata_table: The name of the teradata table to which the data is transferred.(templated)
+            :param aws_conn_id: The Airflow AWS connection used for AWS credentials.
+            :param teradata_conn_id: The connection ID used to connect to Teradata
+                :ref:`Teradata connection <howto/connection:Teradata>`.
+            :param teradata_authorization_name: The name of Teradata Authorization Database Object,
+                is used to control who can access an S3 object store.
+                Refer to
+                https://docs.teradata.com/r/Enterprise_IntelliFlex_VMware/Teradata-VantageTM-Native-Object-Store-Getting-Started-Guide-17.20/Setting-Up-Access/Controlling-Foreign-Table-Access-with-an-AUTHORIZATION-Object
+
+        Examples:
+            .. code-block:: python
+
+                import pandas as pd
+                import pyarrow as pa
+                import pyarrow.parquet as pq
+
+                @op
+                def write_parquet_file(teradata: TeradataResource):
+                    df = pd.DataFrame({"one": [1, 2, 3], "ten": [11, 12, 13]})
+                    table = pa.Table.from_pandas(df)
+                    pq.write_table(table, "example.parquet')
+                    teradata.load_table_from_local_parquet(
```

Comment:
Where load_table_from_local_parquet is defined?

Reply:
This is not required; I will remove this example